### PR TITLE
Update EdgeBridge to major version 4.0, minimum Xcode version to 14.1 and iOS 11. 

### PR DESCRIPTION
**Pull request by cacheung**
## How Has This Been Tested?

To Incorporate that as of April 25th 2023, iOS and iPadOS apps submitted to the App Store must be built with Xcode 14.1.(https://developer.apple.com/news/upcoming-requirements/?id=04252023a)

-Update EdgeBridge to major version 4.0, minimum Xcode version to 14.1 and iOS 11.
-Update SwiftLint version to 0.52
-Update the Functional tests which were complained by too many characters in a sentence.
-Update tutorial app spm versions to 4.0

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -51,17 +54,36 @@ jobs:
...
+    - name: Create GH Release
+      id: create_release
+      uses: release-drafter/release-drafter@v5
       if: ${{ github.event.inputs.action_tag == 'yes' }}
       with:
-        name: v${{ github.event.inputs.tag }}
-        tag: ${{ github.event.inputs.tag }}
-        version: ${{ github.event.inputs.tag }}
-        publish: true
+         name: v${{ github.event.inputs.tag }}
```
Comment (addb) [inline .github/workflows/release.yml:69]: is this indentation correct?

File: AEPEdgeBridge.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 54;
```
Comment (addb) [inline AEPEdgeBridge.xcodeproj/project.pbxproj:6]: Update the Marketing version as well.

File: AEPEdgeBridge.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (addb) [inline AEPEdgeBridge.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:1]: This seems like a new file. Do we need to commit this? @kevinlind @emdobrin

Comment (emdobrin) [reply AEPEdgeBridge.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:1]: Usually xcshareddata settings can be committed as shared settings for the project, but we ignore xcuserdata settings. Is this required for any of the changes you made @cacheung, if not let's revert it

File: Documentation/tutorials/EdgeBridgeTutorialAppFinal/EdgeBridgeTutorialApp/AppDelegate.swift [hunk 4]
```diff
@@ -13,13 +13,12 @@
...
-import AEPEdgeConsent
 import AEPEdgeBridge
+import AEPEdgeConsent
 import AEPEdgeIdentity
 import AEPLifecycle
 
-//* EdgeBridge Tutorial - code section (1/2)
-import AEPEdgeBridge
+// * EdgeBridge Tutorial - code section (1/2)
 // EdgeBridge Tutorial - code section (1/2) */
```
Comment (emdobrin) [inline Documentation/tutorials/EdgeBridgeTutorialAppFinal/EdgeBridgeTutorialApp/AppDelegate.swift:22]: this got reordered after formatting, but we should keep the comment around the import statement for AEPEdgeBridge

**Pull request merged**